src/dataconnect/fileUtils.ts

```diff
@@ -106,3 +106,23 @@ export async function pickService(
   }
   return serviceInfo;
 }
+
+// case insensitive indicators for supported app platforms
+const WEB_INDICATORS = ["package.json", "package-lock.json"];
+const IOS_INDICATORS = ["info.plist", "project.pbxproj", "podfile", "package.swift"];
```

Comment:
@aashishpatil-g to review this list. Based on my minimal knowledge, i think we'd want to look for any file with .xcodeproj or .xcworkspace file extensions.

Reply:
Yes that is correct - look for folders whose names end in suffix .xcodeproj or .xcworkspace 

The project.pbxproj is an internal file within the .xcodeproj package folder, so won't be available at project folder level.

Reply:
Are you suggesting we also check the selected folder's name for .xcodeproj or .xcworkspace?

Reply:
added, please review

Reply:
These will be inside the app folder. In Finder the xcodeproj and .xcworkspace appear as files but at a filesystem level these are folders.

Reply:
to confirm, a user would select an app folder, and within that folder would be an .xcodeproj folder

Reply:
Yes that is correct. Either a folder with suffix .xcodeproj or a folder with suffix .xcworkspace